samples/matter/common/src/diagnostic/log_backend_diagnostic.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "diagnostic_logs_provider.h"
+
+#include <zephyr/logging/log.h>
+#include <zephyr/logging/log_backend.h>
+
+static uint32_t sLogFormat = LOG_OUTPUT_TEXT;
+/* The buffer size was selected to be able to store single log line (usually in Matter we have logs < 200 B).
+ * In case the log will not fit it will be printed anyway, but usng fragmentation, so setting a small buffer
+ * increases the time of printing.
+ */
+static uint8_t sLogBuffer[256];
```

Comment:
Can we use here an existing kconfig instead of raw value?

Reply:
Do you have in mind any specific Kconfig? Other backends just create own Kconfig to control it, but I thought maybe we don't need to provide a way to change the value.

Reply:
No, maybe there is no appropriate one. :/ So, ok let's leave it as it is.